package.json

```diff
@@ -72,10 +72,10 @@
     "@rollup/plugin-image": "2.0.4",
     "@rollup/plugin-node-resolve": "7.1.1",
     "@schematics/angular": "10.0.3",
-    "@storybook/addon-knobs": "5.3.9",
-    "@storybook/angular": "5.3.9",
-    "@storybook/core": "5.3.9",
-    "@storybook/react": "5.3.9",
+    "@storybook/addon-knobs": "6.0.20",
```

Comment:
FYI it's at 6.0.21 already at the time of this comment, may as well make it the latest before merging :)